api/catalog/api/models/audio.py

```diff
@@ -175,6 +176,10 @@ class Audio(AudioFileMixin, AbstractMedia):
         help_text="JSON describing alternative files for this audio.",
     )
 
+    @property
+    def mature(self) -> bool:
+        return MatureAudio.objects.filter(identifier=self.identifier).exists()
+
```

Comment:
~Also occurred to me, could this be moved into the `AbstractMedia` class, given it'll probably need to be implemented for every media type? A `mature_class` property on the implementing class would make this nicely generic.~

Update: Never mind, I see that this is probably not easily or ergonomically possible now due to `Mature_` itself requiring the media model class defined on it. The references would be circular.

Reply:
Based on my understanding, the `@cached_property` decorator wouldn't not do much here, since that it's lifetime is bound to the lifetime of the model instance?

Django has a decorator with the same name, and their [docs](https://docs.djangoproject.com/en/4.1/ref/utils/#django.utils.functional.cached_property) state that the cache only persists as long as the instance is in memory.